src/OSPSuite.Core/Domain/Services/ParameterIdentifications/ResidualCalculator.cs

```diff
@@ -122,5 +130,13 @@ protected double LogSafe(float x)
 
          return Math.Log10(x);
       }
+
+      public ResidualsResult CalculateForSimulation(DataRepository simulationResultRepository, IReadOnlyList<OutputMapping> allOutputMappings)
```

Comment:
Move this method up right below the other one. Public method first

Reply:
done